CHANGELOG.md

```diff
@@ -1,7 +1,10 @@
 ## CHANGELOG
 
-### 3.9.1 (2022-01-20)
+### 4.0.0 (In progress)
+**SDK**
+- **Removed:** Deprecated `getUniqueId` from `MiniAppMessageBridge` and `getAccessToken` from `UserInfoBridgeDispatcher`
```

Comment:
The wording here could be a little confusing. Also, I think we should be specific about the function signature so it doesn't look like we removed the entire feature. How about this instead?
```suggestion
- **Removed:** Functions which were deprecated in previous releases have been removed:
  - `MiniAppMessageBridge.getUniqueId()`
  - `UserInfoBridgeDispatcher.getAccessToken(miniAppId: String, onSuccess: (tokenData: TokenData) -> Unit, onError: (message: String) -> Unit)`
  - `UserInfoBridgeDispatcher.getAccessToken(miniAppId: String, accessTokenScope: AccessTokenScope, onSuccess: (tokenData: TokenData) -> Unit, onError: (message: String) -> Unit)`
```

Reply:
Thank you, I have updated this part.